packages/test/src/integration-tests-old.ts

```diff
@@ -398,7 +398,7 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     await t.throwsAsync(workflow.query('not found'), {
       instanceOf: QueryNotRegisteredError,
       message:
-        'Workflow did not register a handler for not found. Registered queries: [__stack_trace __enhanced_stack_trace isBlocked]',
+        'Workflow did not register a handler for not found. Registered queries: [__stack_trace __enhanced_stack_trace __temporal_workflow_metadata isBlocked]',
```

Comment:
I know it sounds trivial, but can we update the comment at https://github.com/temporalio/api/blob/59df52af8d5ffdd4c2a9748e9711d436c0392aa9/temporal/api/sdk/v1/workflow_metadata.proto#L34 to be `__temporal_workflow_metadata` instead of `__temporal_getWorkflowMetadata`?

Reply:
https://github.com/temporalio/api/pull/342